cmd/copy.go

```diff
@@ -1489,11 +1534,12 @@ func (cca *CookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 		common.EFromTo.BenchmarkFile():
 
 		var e *CopyEnumerator
-		e, err = cca.initEnumerator(jobPartOrder, ctx)
+		srcCredInfo, _ := cca.getSrcCredential(ctx, &jobPartOrder)
```

Comment:
Do we have logs when token refresh is hit so that in future if we see multiple threads doing it in parallel can be debugged from logs itself.